src/main/scala/org/apache/spark/sql/execution/datasources/oap/index/indexPlans.scala

```diff
@@ -76,6 +78,19 @@ case class CreateIndexCommand(
           options = _fsRelation.options)(_fsRelation.sparkSession)
         val logical = LogicalRelation(fsRelation, attributes, id)
         (f, s, OapFileFormat.PARQUET_DATA_FILE_CLASSNAME, id, logical)
+      case LogicalRelation(
+      _fsRelation @ HadoopFsRelation(f, _, s, _, format: OrcFileFormat, _), attributes, id) =>
+        if (!sparkSession.conf.get(OapConf.OAP_ORC_ENABLED)) {
+          throw new OapException(s"turn on ${
+            OapConf.OAP_ORC_ENABLED.key} to allow index building on parquet files")
+        }
+        // Use ReadOnlyParquetFileFormat instead of ParquetFileFormat because of
+        // ReadOnlyParquetFileFormat.isSplitable always return false.
+        val fsRelation = _fsRelation.copy(
+          fileFormat = new ReadOnlyOrcFileFormat(),
+          options = _fsRelation.options)(_fsRelation.sparkSession)
+        val logical = LogicalRelation(fsRelation, attributes, id)
+        (f, s, OapFileFormat.PARQUET_DATA_FILE_CLASSNAME, id, logical)
```

Comment:
code, message and comments include the word "Parquet", we need to change to Orc as well.
 (f, s, OapFileFormat.PARQUET_DATA_FILE_CLASSNAME, id, logical) 
"... allow index building on parquet files"
// Use ReadOnlyParquetFileFormat instead of ParquetFileFormat because of 
// ReadOnlyParquetFileFormat.isSplitable always return false.

Reply:
Yes. Definitely. I will change the comments accordingly.